testConnectivity.py

```diff
@@ -247,13 +250,14 @@ def testConnectivity(args):
         print("Checking GBT Communication (Before Programming GBTs)")
         if not gbtCommIsGood(vfatBoard.parentOH.parentAMC, doReset=True, printSummary=args.debug, ohMask=args.ohMask):
             printRed("Connectivity Testing Failed")
+            printYellow("If Vmon = 8.0V than Imon must be 1.71 +/- 0.01A; if not the GBT's are not locking to the fiber link")
```

Comment:
`than` -> `then`
Is this a QC8 setup specific warning?

Reply:
No, the target voltage for post LS2 operation is 8.0V.

Reply:
I was more interested in the current draw, as we saw quite some variation in V2 current draw, so I was curious how well constrained this value is expected to be

Reply:
Ah I understand...yes in this case the current draw will probably depend on also the length of the LV cable powering the frontend.  I have not accounted for that.